src/main/java/ca/corefacility/bioinformatics/irida/processing/concatenate/impl/SequenceFilePairConcatenator.java

```diff
@@ -21,22 +21,24 @@
 @Component
```

Comment:
If these are just getting created by the `SequencingObjectConcatenatorFactory`, I dont' think they need to be `@Components` since they're not being created by spring.  Can remove the `@Autowired` stuff in these as well.

Reply:
Updated in 97e0014e1c5aa5584ad4b4f07bf67f3d54fbaf70